src/components/PreLoader/PreLoader.ts

```diff
@@ -20,15 +20,15 @@ export class PreLoader implements PreLoaderInterface {
     public async hide() {
         await sleep(HIDE_DURATION);
 
-        this.loader && this.loader.setAttribute("class", "loaded");
+        document.body.setAttribute("class", "loaded");
```

Comment:
body.setAttribute('class') is bad practice because a lot of libraries use body.
We should use `body.classList.add('loaded')` and `body.classList.remove('loaded')` instead.